tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator/ExtensionsMetadataGenerator.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk" InitialTargets="UpdateRuntimeAssemblies">
```

Comment:
Is this required? Doesn't the `BeforeTargets="Build"` handle this?

Reply:
If `runtimeassemblies.txt` file does not exist, build fails without `InitialTargets` . Let me know if I missed anything.